<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleConfig</class>
 <widget class="QWidget" name="StyleConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>363</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Widget Style</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTreeWidget" name="styleList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Toolbar button style:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="toolButtonStyle">
     <item>
      <property name="text">
       <string>Only display the icon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Only display the text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>The text appears beside the icon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>The text appears under the icon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="singleClickActivate">
     <property name="text">
      <string>Activate item on single click</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
